_sass/minimal-mistakes/_navigation.scss

```diff
@@ -571,3 +571,19 @@
     padding-left: 3.25rem
   }
 }
+
+.sidebar__right.sticky > .toc > .toc__menu {
```

Comment:
Also it's going to be ugly on mobile devices where this whole thing is unneeded. Should probably be wrapped inside 

Perhaps [my working code](https://github.com/iBug/iBug-source/blob/cb512482551e516dfe8049c508b5f6550300c337/_sass/toc.scss) ([preview](https://ibug.io/p/23) and [Wayback Machine](https://web.archive.org/web/20210205163606/https://ibug.io/blog/2021/01/linux-container-explained/)) would be a good reference. But keep in mind that those codes are CC BY-SA 4.0 until I actively contribute them to this repo.